lms/static/js/inline_analytics.js

```diff
@@ -378,6 +371,23 @@ window.InlineAnalytics = (function() {
     }
 
 
+    function convertNameToArray(choiceNameString) {
+        // Convert the choice name string to a list
+        choiceNameString = choiceNameString.substring(1, choiceNameString.length -1).replace(/'/g, "");
```

Comment:
Actually, we can drop the 2nd param (endIndex) from the `substring` call, as it defaults to reading until the end of the string (what we're already doing).

```javascript
choiceNameString = choiceNameString.substring(1);
```